src/fu-engine.c

```diff
@@ -3958,26 +3958,31 @@ fu_engine_udev_device_add (FuEngine *self, GUdevDevice *udev_device)
 	/* can be specified using a quirk */
 	plugin_name = fu_device_get_plugin (FU_DEVICE (device));
 	if (plugin_name != NULL) {
-		g_autoptr(GError) error = NULL;
-		FuPlugin *plugin = fu_plugin_list_find_by_name (self->plugin_list,
-								plugin_name, &error);
-		if (plugin == NULL) {
-			g_warning ("failed to find specified plugin %s: %s",
-				   plugin_name, error->message);
-			return;
-		}
-		if (!fu_plugin_runner_udev_device_added (plugin, device, &error)) {
-			if (g_error_matches (error, FWUPD_ERROR, FWUPD_ERROR_NOT_SUPPORTED)) {
-				g_debug ("%s ignoring: %s",
-					 fu_plugin_get_name (plugin),
-					 error->message);
+		g_auto(GStrv) plugins = g_strsplit (plugin_name, ",", -1);
```

Comment:
this is correct here, but won't be supported by the libxmlb version. I'll have a think on how we can handle this kind of case better.